f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -555,6 +555,19 @@ def _merge_members(self, lbaas_members, bigip_members, service):
     @serialized('MemberManager.create')
     @log_helpers.log_method_call
     def create(self, resource, service, **kwargs):
+        create_in_bulk = False
+        if 'multiple' in service:
+            bulk_mode = service.get('multiple')
+            if bulk_mode is True:
+                create_in_bulk = True
```

Comment:
seems no need to use 3 variables . can we just use one to distinguish which mode we are in.

Reply:
yes, I will simplify it. Thanks